File: Cloud5S_WEB/src/app/@filter/MD/partner.filter.ts
import {BaseFilter} from '../Common/base.filter';

export class PartnerFilter extends BaseFilter {
  code!: string;
  name!: string;
  isActive: boolean | string = true;
  address!: string;
  phoneNumber!: string;
  taxCode!: string;
  email!: string;
  isProvider!: boolean | string | null;
  isCustomer!: boolean | string | null;
}
export interface optionsGroup {
  id: string;
  name: string;
}

File: Cloud5S_API/DMS.Business/Filter/MD/PartnerFilter.cs
using DMS.BUSINESS.Filter.Common;

namespace DMS.BUSINESS.Filter.MD
{
    public class PartnerFilter : BaseFilter
    {
        public bool? IsCustomer { get; set; }
        public bool? IsProvider { get; set; }
    }

    public class PartnerFilterLite : BaseMdFilter
    {
        public bool? IsCustomer { get; set; }
        public bool? IsProvider { get; set; }
    }
}
